src/lib.rs

```diff
@@ -62,13 +65,15 @@
 #![no_std]
 #![cfg_attr(feature = "internal_benches", allow(unstable_features), feature(test))]
 
-#[cfg(any(test, feature = "use_heap"))]
+#[cfg(any(test, feature = "alloc"))]
+extern crate alloc;
+
+#[cfg(any(test, feature = "std"))]
 extern crate std;
 
 #[macro_use]
 mod debug;
 
-#[cfg(any(test, feature = "use_heap"))]
 #[macro_use]
```

Comment:
This still needs a `#[cfg(test)]` in order for `cargo build --no-default-features` to work.

Reply:
Actually, `#[cfg(feature = "alloc")]` is what you want here, so that the integration tests keep working.